ground/gcs/gcs.pri

```diff
@@ -57,8 +57,8 @@ isEmpty(GCS_BUILD_TREE) {
 }
 GCS_APP_PATH = $$GCS_BUILD_TREE/bin
 macx {
-    QMAKE_CFLAGS_X86_64 += -mmacosx-version-min=10.7
-    QMAKE_CXXFLAGS_X86_64 = $$QMAKE_CFLAGS_X86_64
+#    QMAKE_CFLAGS_X86_64 += -mmacosx-version-min=10.7

```

Comment:
We don't really like commented code. If this isn't needed please trash these lines.